### ISSUE TITLE
Can't find module "caching"

### ISSUE THREAD

**Issue by ERmilburn02** [AuroraFN/backend#1]
Get this error when running

```
PS C:\Users\ERmil\Aurora\backend> node .\index.js
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module 'C:\Users\ERmil\Aurora\backend/structs/caching'
Require stack:
- C:\Users\ERmil\Aurora\backend\index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ERmil\Aurora\backend\index.js:32:1)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'C:\\Users\\ERmil\\Aurora\\backend\\index.js' ]
}
```

was a file missed?

**Comment by Slushia** [comment]
Fixed

**Comment by AyeTSG** [comment]
While yes the original issue is fixed, this now throws another error related to caching.

```
pi@raspberrypi:~/src/github_repos/AuroraFN/backend $ node index.js
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '/home/pi/src/github_repos/AuroraFN/backend/routes/../structs/cachi                                                                                                ng'
Require stack:
- /home/pi/src/github_repos/AuroraFN/backend/routes/id.js
- /home/pi/src/github_repos/AuroraFN/backend/routes/index.js
- /home/pi/src/github_repos/AuroraFN/backend/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/home/pi/src/github_repos/AuroraFN/backend/routes/id.js:11:17)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/home/pi/src/github_repos/AuroraFN/backend/routes/index.js:23:16)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/pi/src/github_repos/AuroraFN/backend/routes/id.js',
    '/home/pi/src/github_repos/AuroraFN/backend/routes/index.js',
    '/home/pi/src/github_repos/AuroraFN/backend/index.js'
  ]
}
```

**Comment by AyeTSG** [comment]
@Slushia AFAIK the repo appears to be missing ``/structs/caching.js``

**Comment by Slushia** [comment]
Sorry, I guess the cache file never was pushed correctly? Should be fixed in [#83b3cc8](https://github.com/AuroraFN/backend/commit/83b3cc80579d5fb344b6c69b00d55048a2b612ca)

**Comment by AyeTSG** [comment]
Can confirm this was fixed with [#83b3cc8](https://github.com/AuroraFN/backend/commit/83b3cc80579d5fb344b6c69b00d55048a2b612ca), thanks!
